app/models/option.rb

```diff
@@ -0,0 +1,4 @@
+class Option < ApplicationRecord
+  belongs_to :poll
+  validates_presence_of :value, :votes, :poll_id
```

Comment:
Rename votes to something else. Don't share 'model' names. It creates confusion. Both to reader as well as Rails (because Rails works via convention)